e2e-tests/spark-apps/spark_3_4_1/spark-pi_scala_2.12.yaml

```diff
@@ -0,0 +1,33 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+#
+
+apiVersion: org.apache.spark/v1alpha1
+kind: SparkApplication
+metadata:
+  name: spark-pi-341-212
```

Comment:
ditto.

Reply:
FYI, this is not the latest even in Spark 3.4.x line because we have 3.4.2 already. Let's use 3.5.1.

Reply:
Ack - the `e2e-tests` dir aims to include examples for all currently supported Spark versions. I started this effort with [all available versions with official image](https://hub.docker.com/_/spark) . I'll replace the examples to remove old versions to avoid confusing people.